task-launcher/src/tasks/shared/trials/taskFinished.js

```diff
@@ -2,33 +2,39 @@ import jsPsychHTMLMultiResponse from '@jspsych-contrib/plugin-html-multi-respons
 import { taskStore, PageAudioHandler } from '../helpers';
 import { mediaAssets } from '../../..';
 
-export const taskFinished = {
-  type: jsPsychHTMLMultiResponse,
-  data: () => {
-    return {
-      // save_trial: true,
-      assessment_stage: 'instructions',
-    };
-  },
-  stimulus: () => {
-    const t = taskStore().translations;
-    return `
-        <div class='lev-stimulus-container'>
-          <div class='lev-row-container instruction'>
-            <h1>${t.taskFinished}</h1>
-          </div>
-            <footer>${t.generalFooter}</footer>
-        </div>`;
-  },
-  button_choices: [`Continue`],
-  keyboard_choices: 'ALL_KEYS',
-  button_html: '<button class="primary" style=margin-top:10%>Exit</button>',
-  on_load: () => {
-    if (mediaAssets.audio.taskFinished) {
-      PageAudioHandler.playAudio(mediaAssets.audio.taskFinished)
+export const taskFinished = (intro = false) => {
+  return {
+    type: jsPsychHTMLMultiResponse,
+    data: () => {
+      return {
+        // save_trial: true,
+        assessment_stage: 'instructions',
+      };
+    },
+    stimulus: () => {
+      const t = taskStore().translations;
+      const endMessage = intro ? 'introFinished' : 'taskFinished'; 
+
+      return `
+          <div class='lev-stimulus-container'>
+            <div class='lev-row-container instruction'>
+              <h1>${t[endMessage]}</h1>
+            </div>
+              <footer>${t.generalFooter}</footer>
+          </div>`;
+    },
+    button_choices: [`Continue`],
+    keyboard_choices: 'ALL_KEYS',
+    button_html: '<button class="primary" style=margin-top:10%>Exit</button>',
+    on_load: () => {
+      const endMessage = intro ? 'introFinished' : 'taskFinished';
```

Comment:
Can we rename the mediaAsset to `taskFinished` instead of `introFinished`?

Reply:
`taskFinished` is the asset that all the other tasks use - the text is "You've completed the game. Thank you!" and `introFinished` is a separate asset for intro only that I just created (it says "You've finished the instructions. Thank you!"). I could definitely rename `introFinished`, but I think it has to be different from `taskFinished`. Sorry if I'm misunderstanding your comment though!

Reply:
Makes sense. I guess when we pull them both in the common folder that we have been talking about, both can exist. So a config param makes sense. I am wondering though for future proofing should we pass the asset name as the parameter to `taskFinished` as opposed to supporting just two use cases. If we pass the asset name then in the future we can support as many ending screens as we want. But that is optional, I'll leave it up to you.

Reply:
That makes sense, I switched it so that we now pass in the asset name.